changelogs/fragments/265-fix-element-type-ecs_service.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - ecs_service - fix element type for load_balancers parameter (https://github.com/ansible-collections/community.aws/issues/265)
```

```suggestion
  - ecs_service - fix element type for load_balancers parameter (https://github.com/ansible-collections/community.aws/issues/265).
```
Same for the other PR :)